integration/flink/src/main/java/org/apache/carbon/flink/CarbonS3Property.java

```diff
@@ -27,6 +27,7 @@
 
   static final String DATA_TEMP_PATH = "carbon.writer.s3.data.temp.path";
 
+  @Deprecated
```

Comment:
I think we can delete this constant directly, since this version has not been released

Reply:
deleted.